feat: extension move

edgarkhanzadian opened the pull request:
> Try out Leather build 59a58c3 — [Extension build](https://github.com/leather-io/mono/actions/runs/18413403905), [Test report](https://leather-io.github.io/playwright-reports/feat/extension-move), [Storybook](https://feat/extension-move--65982789c7e2278518f189e7.chromatic.com), [Chromatic](https://www.chromatic.com/library?appId=65982789c7e2278518f189e7&branch=feat/extension-move)<!-- Sticky Header Marker -->

Using this [guide](https://developers.netlify.com/guides/migrating-git-from-multirepo-to-monorepo-without-losing-history/) and [git-filter-repo](https://github.com/newren/git-filter-repo) i was able to bring extension + history into monorepo.

Keeping steps for execution here as a future reference

Ready for merge as soon as #1630 merges

Steps for extension:

```sh
git clone https://github.com/leather-io/extension
cd extension

# make sure you're on dev branch
git checkout dev

# move all of the files + update the commits to a subdirectory which matches a monorepo subdirectory
git filter-repo --to-subdirectory-filter apps/extension

# Update all of the commits/issues/prs in the commit history of leather-io/extension. Now, instead of "closes #X" it would be "closes (leather-io/extension#X)"
# This is a modified version of a command that is provided by the guide
git filter-repo --commit-callback '
    import re
    msg = commit.message.decode("utf-8")
    pattern = r"(?<![/\w])((?:PR |fix |fixes |close |closes |ref )?)#(\d+)"
    replacement = r"\g<1>(leather-io/extension#\g<2>)"
    newmsg = re.sub(pattern, replacement, msg)
    commit.message = newmsg.encode("utf-8")
    '

# for squashing commits past sept1 2023
FIRST_COMMIT=dc12485c6f499dbfc92f1a7c57928798818c7d55
NEXT_COMMIT=8beb1929957b0f603ec951cf259596593c0ee961
NEW_ROOT=$(
    GIT_AUTHOR_NAME="leather-bot" \
        GIT_AUTHOR_EMAIL="leather-bot@leather.io" \
        GIT_COMMITTER_NAME="leather-bot" \
        GIT_COMMITTER_EMAIL="leather-bot@leather.io" \
        GIT_AUTHOR_DATE="$(git log -1 --format=%aD $FIRST_COMMIT)" \
        GIT_COMMITTER_DATE="$(git log -1 --format=%cD $FIRST_COMMIT)" \
        git commit-tree -m "chore(release): 6.5.1" $FIRST_COMMIT^{tree}
)
git replace $FIRST_COMMIT $NEW_ROOT
# point commits to NEW_ROOT
git filter-repo --commit-callback "
if commit.original_id == b'$NEXT_COMMIT':
    commit.parents = [b'$NEW_ROOT']
"
# Force changes and clean up
git filter-repo --force

```

Steps for monorepo:

```sh

git clone https://github.com/leather-io/mono

cd mono

git checkout -b feat/extension-move

git remote add temp ../extension

git fetch temp

git merge temp/dev --allow-unrelated-histories

git push -u origin feat/extension-move --force


```

tigranpetrossian committed:
refactor: address breaking changes in Avatar

tigranpetrossian committed:
refactor: address breaking changes in Avatar

kyranjamie committed:
feat: network param support, getAddresses, closes LEA-2195

Commit:
refactor: rpc send transfer

Commit:
refactor: send transfer fees to shared fee editor

kyranjamie committed:
fix: improve account look up speed, ref LEA-2133

kyranjamie committed:
chore: conditional logging of tx montior events

kyranjamie committed:
Merge pull request (leather-io/extension#6163) from leather-io/release/monday-fixes

semantic-release-bot committed:
chore(release): 6.65.0

## [6.65.0](https://github.com/leather-io/extension/compare/v6.64.0...v6.65.0) (2025-03-03)

### Features

* implement stacks fee estimations, LEA-1760 ([ea5fe9a](https://github.com/leather-io/extension/commit/ea5fe9a2abe0a58ca4456ad9547eee6f494115f0))
* network param support, getAddresses, closes LEA-2195 ([7451fc6](https://github.com/leather-io/extension/commit/7451fc666b5467b5dacd09529f0f1c6071b327f1))
* stx transfer sip9 nft, closes LEA-1964 ([4cb2b44](https://github.com/leather-io/extension/commit/4cb2b442cbd871a724880ea166db90b2a897db90))

### Bug Fixes

* broadcasting to default stacks network ([c9a50e0](https://github.com/leather-io/extension/commit/c9a50e0f49f2fb4da9750d9baab8c64dee059f2c))
* btc recipient validation, closes LEA-2126 ([ad77a40](https://github.com/leather-io/extension/commit/ad77a400efbcfbf8f32932e4aa4cfd3cfd4dc7fa))
* improve account look up speed, ref LEA-2133 ([f33d4a4](https://github.com/leather-io/extension/commit/f33d4a4ca6629d9de32f013eaa9a16f374a119ad))
* issue with broken summary page, closes [(leather-io/extension#6152)](https://github.com/leather-io/extension/issues/6152) ([28cb7f0](https://github.com/leather-io/extension/commit/28cb7f00583265afc2b3c75cd8fe20068d394baf))
* **ledger:** switching accounts ([4fd462b](https://github.com/leather-io/extension/commit/4fd462b126bac54857fef54b785e69a26fde68bd))
* sbtc limits schema, closes LEA-2185 ([019eef3](https://github.com/leather-io/extension/commit/019eef34f55a97a883013fac58e80101770a6857))
* unhandled err in stx send form, LEA-2140 ([b0d3731](https://github.com/leather-io/extension/commit/b0d3731b9e50428e67aaddb635b42d378e115161))
* zero nonce not getting set, closes LEA-2201 ([cfd808e](https://github.com/leather-io/extension/commit/cfd808e894a3cbe6623d5a9adbd14441f6003423))

### Internal

* remove unnecessary action type ([e079891](https://github.com/leather-io/extension/commit/e07989168564b358c2de4f0e43d6d5edecdbba54))
* update bitcoin, stacks, query and [@scure](https://github.com/scure) packages ([ca788f7](https://github.com/leather-io/extension/commit/ca788f7e6e7d74fe8f297c22c4ec8b75f955189c))
* use sbtc lib deposit defaults ([7eede9d](https://github.com/leather-io/extension/commit/7eede9d765c2790b5399684109519f96c2f17b9f))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix: default to port 8000 for devnet, closes (leather-io/extension#2174)

alter-eggo committed:
fix: unhandled stx tx broadcast error, LEA-2203

Commit:
refactor: use loaders and remove effect hooks

kyranjamie committed:
fix(rpc): prevent opening on fail

kyranjamie committed:
fix: error during formating causing broadcast error

kyranjamie committed:
fix: support networks on tx calls

kyranjamie committed:
ci: behaviours for needs: labels

Commit:
refactor: address fee editor naming concerns

kyranjamie committed:
refactor(rpc): register rpc request handlers

kyranjamie committed:
fix: improve copy

Co-authored-by: Will Cameron <camerow@users.noreply.github.com>

Commit:
refactor: stx call contract to approver, closes LEA-2170

Commit:
fix: sip30 success response, closes (leather-io/extension#6152)

kyranjamie committed:
ci: production deploy environment

alter-eggo committed:
chore: update leather packages

kyranjamie committed:
ci: audit all commits are signed on pr

Commit:
fix: psbt null output address

kyranjamie committed:
Merge pull request (leather-io/extension#6178) from leather-io/release/tuesday-one-pm

Release/tuesday one pm

semantic-release-bot committed:
chore(release): 6.65.1

## [6.65.1](https://github.com/leather-io/extension/compare/v6.65.0...v6.65.1) (2025-03-12)

### Bug Fixes

* default to port 8000 for devnet, closes [(leather-io/extension#2174)](https://github.com/leather-io/extension/issues/2174) ([7f62f25](https://github.com/leather-io/extension/commit/7f62f25c409d7554d01ea40aabf4487b75955faf))
* error during formating causing broadcast error ([c5848dd](https://github.com/leather-io/extension/commit/c5848dd702d5fea82337c0037baa48d1358dff3f))
* improve copy ([8f31caa](https://github.com/leather-io/extension/commit/8f31caa522f6688ae6c4e414c2e499cf995f1f32))
* psbt null output address ([33f1b24](https://github.com/leather-io/extension/commit/33f1b2464e3ac49fa8a38209f934d34d3efeea53))
* **rpc:** prevent opening on fail ([7851972](https://github.com/leather-io/extension/commit/7851972737a433973434c9008fbff7ec0a753501))
* sip30 success response, closes [(leather-io/extension#6152)](https://github.com/leather-io/extension/issues/6152) ([d05c96c](https://github.com/leather-io/extension/commit/d05c96c648b91c27d7b2e5b6eec36692d310f49c))
* support networks on tx calls ([3811433](https://github.com/leather-io/extension/commit/3811433008e9fa8ce15d9ea90c62ff059228d3ef))
* unhandled stx tx broadcast error, LEA-2203 ([f092c09](https://github.com/leather-io/extension/commit/f092c09fc7923c027920d1818d6d29cad529da38))

### Internal

* address fee editor naming concerns ([9dabbc9](https://github.com/leather-io/extension/commit/9dabbc94e01726cd45c5b9e33833f701ca6304ec))
* conditional logging of tx montior events ([cca2c11](https://github.com/leather-io/extension/commit/cca2c110929bb498250514c5c3b74bf356068737))
* post-release merge back ([b596fab](https://github.com/leather-io/extension/commit/b596fab09e652862b38ec79dbe06c03b5a9179ee))
* rpc send transfer ([d393eac](https://github.com/leather-io/extension/commit/d393eac3efe27a53c0e151ad1d34bdb568668098))
* **rpc:** register rpc request handlers ([2113a0c](https://github.com/leather-io/extension/commit/2113a0c84aaf253873f89c6854e10e321b9bca25))
* send transfer fees to shared fee editor ([7330c97](https://github.com/leather-io/extension/commit/7330c9718894d003ae3a020f1c9cf6102396d054))
* update leather packages ([b07bade](https://github.com/leather-io/extension/commit/b07badeebdad4e17f130329e02957d8d1b4f4791))
* use loaders and remove effect hooks ([ba3e482](https://github.com/leather-io/extension/commit/ba3e4826b9cda624b871293a568b19941f256cbe))

leather-bot committed:
chore: post-release merge back

camerow committed:
fix: update analytics usage

refactor: replace analytics client with segment client in event decoration

camerow committed:
fix: update analytics usage

refactor: replace analytics client with segment client in event decoration

camerow committed:
fix: test ci fix

camerow committed:
fix: test ci fix

Commit:
fix: remove skipping utxos check unless test env

Commit:
fix: remove skipping utxos check unless test env

Commit:
fix: provide feedback to user while broadcasting, closes (leather-io/extension#6193)

Commit:
fix: provide feedback to user while broadcasting, closes (leather-io/extension#6193)

Commit:
fix: settings test

Commit:
fix: settings test

Commit:
Merge pull request (leather-io/extension#6197) from leather-io/release/fix-send-btc

release: fix send btc

semantic-release-bot committed:
chore(release): 6.65.2

## [6.65.2](https://github.com/leather-io/extension/compare/v6.65.1...v6.65.2) (2025-03-25)

### Bug Fixes

* provide feedback to user while broadcasting, closes [(leather-io/extension#6193)](https://github.com/leather-io/extension/issues/6193) ([2fe81b9](https://github.com/leather-io/extension/commit/2fe81b978e0d09f89ab4402b477261ff06497668))
* remove skipping utxos check unless test env ([d6a1f62](https://github.com/leather-io/extension/commit/d6a1f62910a2189ee9a731a1ca2a140958fc8485))
* settings test ([c4573e6](https://github.com/leather-io/extension/commit/c4573e6944af4f57a694e5ebb268c432be6627be))
* test ci fix ([ecdf440](https://github.com/leather-io/extension/commit/ecdf440d8329133cb669fd1917101deeb1f870c9))
* update analytics usage ([2a57470](https://github.com/leather-io/extension/commit/2a5747046bd1a13456a32b0b3b26810b1c935108))

### Internal

* address breaking changes in Avatar ([d834a6d](https://github.com/leather-io/extension/commit/d834a6d4e0b2c981b30f806edc8c6b3eb507c23d))
* post-release merge back ([079f5e0](https://github.com/leather-io/extension/commit/079f5e07bc151dbc51de85e81e80887025ab26d0))

leather-bot committed:
chore: post-release merge back

edgarkhanzadian committed:
feat: move queries back from @leather.io/query

Commit:
refactor: hiro deprecated endpoints, closes LEA-2368

Commit:
fix: utxo inscription check, closes LEA-2398

kyranjamie committed:
feat: open fullpage api, closes LEA-2436

Commit:
Revert "refactor: stx call contract to approver, closes LEA-2170"

This reverts commit 8ba699283d1c0b11009d3aaa2a2e71597dc6ef74.

Commit:
feat: stx call contract with approver, closes LEA-2450

This reverts commit 13afbd9ec914efc70af53544253b7bfc6e190ba8.

Commit:
feat: stx call contract with approver, closes LEA-2450

This reverts commit 13afbd9ec914efc70af53544253b7bfc6e190ba8.

Commit:
fix: utxo check in send transfer

kyranjamie committed:
refactor: use consistent request naming, m̵e̵s̵s̵a̵g̵e̵

kyranjamie committed:
refactor: use helper for popup closer listner, response

314159265359879 committed:
refactor: update wallet config, closes LEA-1307

add some contract addresses and runes

Signed-off-by: Werner <33360391+314159265359879@users.noreply.github.com>

Commit:
refactor: remove stacks transactions v6, closes ENG-10

Commit:
Merge pull request (leather-io/extension#6209) from leather-io/release/new-hiro-apis

release: new hiro apis

semantic-release-bot committed:
chore(release): 6.66.0

## [6.66.0](https://github.com/leather-io/extension/compare/v6.65.2...v6.66.0) (2025-04-07)

### Features

* move queries back from [@leather](https://github.com/leather).io/query ([188aa04](https://github.com/leather-io/extension/commit/188aa04ec0a87e8cc89057de35cc8591f788e622))
* open fullpage api, closes LEA-2436 ([ba64f81](https://github.com/leather-io/extension/commit/ba64f8178199a90571186e97e70f757d6480d7c1))

### Bug Fixes

* provide feedback to user while broadcasting, closes [(leather-io/extension#6193)](https://github.com/leather-io/extension/issues/6193) ([7b211ce](https://github.com/leather-io/extension/commit/7b211ce0da9f4cb35042180f6371ebed1ea3b128))
* remove skipping utxos check unless test env ([dabbb57](https://github.com/leather-io/extension/commit/dabbb573c819302a9cbfe7c700a5e5ac956dfe62))
* settings test ([ccbbb7b](https://github.com/leather-io/extension/commit/ccbbb7b6e4d452cba6b2e410dd87089f154b1a80))
* test ci fix ([12848df](https://github.com/leather-io/extension/commit/12848df6904a9ca35245b9519b33d8e48ca88a2b))
* update analytics usage ([b5493cf](https://github.com/leather-io/extension/commit/b5493cfccf3fb67315b8fe25a4c489ab47a19d02))
* utxo check in send transfer ([961c8bb](https://github.com/leather-io/extension/commit/961c8bb24d552067f14a6552c92f5427ef4a9e8d))
* utxo inscription check, closes LEA-2398 ([843c9e6](https://github.com/leather-io/extension/commit/843c9e6f2b7805105df8b196e6c6b8343d16f92a))

### Internal

* address breaking changes in Avatar ([9a21925](https://github.com/leather-io/extension/commit/9a2192585319bd94a26dea9127abdb537b00e64d))
* hiro deprecated endpoints, closes LEA-2368 ([fc02e38](https://github.com/leather-io/extension/commit/fc02e387e05dba18ee0a7218093fe9d5ca40aff9))
* post-release merge back ([0c1493a](https://github.com/leather-io/extension/commit/0c1493aab7ba5a1c5292ba380a07268a2be4827d))
* remove stacks transactions v6, closes ENG-10 ([75de0f1](https://github.com/leather-io/extension/commit/75de0f1078928a122e2d1b308aab31f8f7f31a46))
* update wallet config, closes LEA-1307 ([24855e4](https://github.com/leather-io/extension/commit/24855e4cfbaa5f692bf76afaa86420c170a71c05))
* use consistent request naming, m̵e̵s̵s̵a̵g̵e̵ ([a4b3ce9](https://github.com/leather-io/extension/commit/a4b3ce95f7804c58abcf02680719ffa911f945a7))
* use helper for popup closer listner, response ([f50581d](https://github.com/leather-io/extension/commit/f50581d2ece7f610c52956e847b540914db72c3b))

leather-bot committed:
chore: post-release merge back

Commit:
fix: minor padding issue with pc

Commit:
fix: minor padding issue with pc

Commit:
feat: legacy request event tracking, closes LEA-2233

Commit:
feat: legacy request event tracking, closes LEA-2233

Commit:
fix: close window on sip30 broadcast success

Commit:
refactor: sip30 transfer stx request to approver, closes LEA-2453

Commit:
fix: close window on sip30 broadcast success

kyranjamie committed:
fix: print amount, 0.1 appeared as 0

kyranjamie committed:
fix: broadcast not including hiro header

kyranjamie committed:
fix: prefer account index app connected with

semantic-release-bot committed:
chore(release): 6.67.0

## [6.67.0](https://github.com/leather-io/extension/compare/v6.66.0...v6.67.0) (2025-04-22)

### Features

* legacy request event tracking, closes LEA-2233 ([82416a6](https://github.com/leather-io/extension/commit/82416a65b4b9a3ebcb97f9fe8d5727e4438f96aa))
* stx call contract with approver, closes LEA-2450 ([83b5680](https://github.com/leather-io/extension/commit/83b568023c6a5b2f4491e65a333ca45b568dd8df))

### Bug Fixes

* close window on sip30 broadcast success ([4822a32](https://github.com/leather-io/extension/commit/4822a32cbf9418ba6b88117e13c71a500326522a))
* minor padding issue with pc ([d8d354c](https://github.com/leather-io/extension/commit/d8d354cf56e71c2033fb6297d0693ac00e83b80f))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix: chrome rejection, sentry issue

kyranjamie committed:
Merge pull request (leather-io/extension#6226) from leather-io/release/thursday-am

semantic-release-bot committed:
chore(release): 6.68.0

## [6.68.0](https://github.com/leather-io/extension/compare/v6.67.0...v6.68.0) (2025-04-24)

### Features

* legacy request event tracking, closes LEA-2233 ([248fc50](https://github.com/leather-io/extension/commit/248fc503c97a4012ceba4e91c251f0faf8d637df))
* stx call contract with approver, closes LEA-2450 ([3f9e2d0](https://github.com/leather-io/extension/commit/3f9e2d0d447f1a872c60196833fb29129647efbd))

### Bug Fixes

* chrome rejection, sentry issue ([6759ad4](https://github.com/leather-io/extension/commit/6759ad4217e6cf6506d9e08c861070ce1a58710e))
* close window on sip30 broadcast success ([152e7ea](https://github.com/leather-io/extension/commit/152e7ea4576dfaf38d1a67064ec24f6f3463f87b))
* minor padding issue with pc ([37bb14b](https://github.com/leather-io/extension/commit/37bb14bfe226f26433903eba04f987be2e53f122))

### Internal

* post-release merge back ([33e149b](https://github.com/leather-io/extension/commit/33e149bcf334b5b6d859933b6bc54be04187a576))
* post-release merge back ([6a86130](https://github.com/leather-io/extension/commit/6a86130dfafc4b2f267a4dfe6099ff3f8aac1202))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix(ledger): incorrect chain signer for stacks flow, closes (leather-io/extension#6223), LEA-63

Commit:
feat: ledger route gates

Commit:
fix: wrong asset ids in wallet config, closes ENG-31

Commit:
fix: return children if not ledger

Commit:
fix: has ledger keys

Commit:
Merge pull request (leather-io/extension#6231) from leather-io/release/ledger-fixes

Release/ledger fixes

semantic-release-bot committed:
chore(release): 6.69.0

## [6.69.0](https://github.com/leather-io/extension/compare/v6.68.0...v6.69.0) (2025-04-28)

### Features

* ledger route gates ([0e0bc7c](https://github.com/leather-io/extension/commit/0e0bc7cdbc186083852dbc1364fb85f893a4701f))

### Bug Fixes

* broadcast not including hiro header ([3bc3109](https://github.com/leather-io/extension/commit/3bc3109449879d1b45f7ba41d7bafaa0f033a1d8))
* has ledger keys ([2e95284](https://github.com/leather-io/extension/commit/2e95284c9e7386da0992eca1f1cd8af029842449))
* **ledger:** incorrect chain signer for stacks flow, closes [(leather-io/extension#6223)](https://github.com/leather-io/extension/issues/6223), LEA-63 ([21ee657](https://github.com/leather-io/extension/commit/21ee65772ad7388f3b0e9ec72cdb533b315945c5))
* prefer account index app connected with ([0564c25](https://github.com/leather-io/extension/commit/0564c25b54c22e282af9c3bdf671e94401ae2dc1))
* print amount, 0.1 appeared as 0 ([e2b5330](https://github.com/leather-io/extension/commit/e2b533093380df83d8ce9ebd9a6d0d67ec8f1267))
* return children if not ledger ([9a54afb](https://github.com/leather-io/extension/commit/9a54afbaae1c1390f38e2adec2c7ca0386e0a8a1))

### Internal

* post-release merge back ([26ab631](https://github.com/leather-io/extension/commit/26ab6311449bc7706cf22a07ac8a3b4da3cdc548))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix: chrome rejection, segment issue

pete-watters committed:
fix(web): use a local resource for sbtc logo image, ref LEA-2551

kyranjamie committed:
ci: don't run signed checks on release

Commit:
Merge pull request (leather-io/extension#6235) from leather-io/release/remove-remote-code

chore: post-release merge back

semantic-release-bot committed:
chore(release): 6.69.1

## [6.69.1](https://github.com/leather-io/extension/compare/v6.69.0...v6.69.1) (2025-04-30)

### Bug Fixes

* chrome rejection, segment issue ([8177d9a](https://github.com/leather-io/extension/commit/8177d9a67e102276262beb9dca3d19e4f948032c))
* **web:** use a local resource for sbtc logo image, ref LEA-2551 ([ea7787a](https://github.com/leather-io/extension/commit/ea7787a7002baf0dc597d67313f6467aa1d2a5a5))
* wrong asset ids in wallet config, closes ENG-31 ([41c4326](https://github.com/leather-io/extension/commit/41c43269d4c50c0afd11eda6cffe888932ea79b9))

### Internal

* post-release merge back ([ed24229](https://github.com/leather-io/extension/commit/ed242294a982677d19214480b58e835aec732260))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix: remove remote loaded code entirely

kyranjamie committed:
fix: prevent signed commits warning

kyranjamie committed:
Merge pull request (leather-io/extension#6238) from leather-io/release/remove-remote-code-v2

semantic-release-bot committed:
chore(release): 6.69.2

## [6.69.2](https://github.com/leather-io/extension/compare/v6.69.1...v6.69.2) (2025-05-01)

### Bug Fixes

* prevent signed commits warning ([47b84c8](https://github.com/leather-io/extension/commit/47b84c81e801f7e7b5e638721ccf341b11dde640))
* remove remote loaded code entirely ([90193e8](https://github.com/leather-io/extension/commit/90193e865e5531c79ef23100649251546918cedf))

### Internal

* post-release merge back ([fd26a23](https://github.com/leather-io/extension/commit/fd26a23fa6019964947a00d3a73ff9e17049574a))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix(rpc): remove default account for 'getAddresses' flow

kyranjamie committed:
ci: add astrix to glob

kyranjamie committed:
Merge pull request (leather-io/extension#6242) from leather-io/release/remove-remote-code-v3

semantic-release-bot committed:
chore(release): 6.69.3

## [6.69.3](https://github.com/leather-io/extension/compare/v6.69.2...v6.69.3) (2025-05-01)

### Bug Fixes

* **rpc:** remove default account for 'getAddresses' flow ([86be9e6](https://github.com/leather-io/extension/commit/86be9e6f2577acd615330788ebc583683e782e12))

### Internal

* post-release merge back ([490ca6c](https://github.com/leather-io/extension/commit/490ca6caec85936761e6f4e3ad72c7b82786a7a8))

leather-bot committed:
chore: post-release merge back

Commit:
refactor: stx transfer sip10 ft to approver, closes LEA-2454

kyranjamie committed:
fix: remove dead coincap api

Commit:
refactor: stx transfer sip9 nft to approver, closes LEA-2455

Commit:
refactor: stx deploy contract to approver, closes LEA-2604 and ENG-68

Commit:
refactor: update call contract

Commit:
refactor: cleanup rpc request implementation, closes LEA-2614

Commit:
refactor: stx sign transaction to approver, closes LEA-2613 and LEA-2617

Commit:
fix: update bitflow sdk, closes LEA-2633

Commit:
feat: web app promo card banner

Commit:
fix: turn off promo card for release

Commit:
fix: test adding switch account to call contract

Commit:
Merge pull request (leather-io/extension#6262) from leather-io/release/bitflow-sdk-upgrade

release: bitflow sdk upgrade

semantic-release-bot committed:
chore(release): 6.70.0

## [6.70.0](https://github.com/leather-io/extension/compare/v6.69.3...v6.70.0) (2025-05-20)

### Features

* web app promo card banner ([d2b02ac](https://github.com/leather-io/extension/commit/d2b02ac25f095066d5779aabf2a61ac3bf1a0af2))

### Bug Fixes

* remove dead coincap api ([1252ea0](https://github.com/leather-io/extension/commit/1252ea0542a365653d9e6eaca174481849bb7216))
* turn off promo card for release ([4cfd33a](https://github.com/leather-io/extension/commit/4cfd33ab1fda7d2866490a7795b2e718b37e06e0))
* update bitflow sdk, closes LEA-2633 ([6f080a5](https://github.com/leather-io/extension/commit/6f080a5ad5abe14f12a9fa3bee686a2a968a733c))

### Internal

* cleanup rpc request implementation, closes LEA-2614 ([c538f02](https://github.com/leather-io/extension/commit/c538f02c520e879cfd49e3a548eebeee9c0340df))
* post-release merge back ([2e30e13](https://github.com/leather-io/extension/commit/2e30e1330505392bd9bd1615ca43e8b977ff7c8c))
* sip30 transfer stx request to approver, closes LEA-2453 ([1a5f053](https://github.com/leather-io/extension/commit/1a5f05372eb2606cbc99512ead558ca24302bae7))
* stx deploy contract to approver, closes LEA-2604 and ENG-68 ([808b443](https://github.com/leather-io/extension/commit/808b443b0b8b9a0019d2313ff53066c65ef66811))
* stx transfer sip10 ft to approver, closes LEA-2454 ([c31f7e7](https://github.com/leather-io/extension/commit/c31f7e7cbe131a4c37ca3b2bbcb2e5f6fc3b2ef6))
* stx transfer sip9 nft to approver, closes LEA-2455 ([ad92a0f](https://github.com/leather-io/extension/commit/ad92a0f96495e4a9ed9e7b5157fc3daed148b13e))
* update call contract ([4fb95c0](https://github.com/leather-io/extension/commit/4fb95c06314b57d2b1c81c24ff7d74436df48367))

leather-bot committed:
chore: post-release merge back

Commit:
chore: enable web app banner

Commit:
Merge pull request (leather-io/extension#6269) from leather-io/chore/enable-web-app-banner

chore: enable web app banner

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix: ability to discard inscription

kyranjamie committed:
fix: release discard

semantic-release-bot committed:
chore(release): 6.70.1

## [6.70.1](https://github.com/leather-io/extension/compare/v6.70.0...v6.70.1) (2025-05-22)

### Bug Fixes

* ability to discard inscription ([575e174](https://github.com/leather-io/extension/commit/575e17412f6f669b91dc097692ce72a5dede5b93))
* release discard ([efe7020](https://github.com/leather-io/extension/commit/efe7020cdc04ec4c3eb52b20f19d900d7d8565c7))

### Internal

* enable web app banner ([b59a0d7](https://github.com/leather-io/extension/commit/b59a0d73bd3945c6a1b70c125b7c8b5d56f3a90e))
* post-release merge back ([161c9a4](https://github.com/leather-io/extension/commit/161c9a479b4ff26e230a6d163e128652cb0a493c))
* post-release merge back ([dc35e67](https://github.com/leather-io/extension/commit/dc35e67863435d7c30ef02fcc11644581096d33d))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
feat: additional inscription controls

kyranjamie committed:
fix: upgrade to node 22

Commit:
refactor: validate send to contract address, closes LEA-2125

Commit:
refactor: account signer, closes LEA-2638

kyranjamie committed:
chore: add deeper tx analytics

Commit:
chore: upgrade leather packages

kyranjamie committed:
chore: add back Flag prop removed from lib

kyranjamie committed:
fix: stacks tx times, closes LEA-2703

Commit:
chore: update React to v19, closes LEA-2709

Commit:
fix: rpc sponsored txs, closes ENG-98

Commit:
fix: rpc flow sbtc sponsorship, closes ENG-97

kyranjamie committed:
ci: remove package update job

kyranjamie committed:
fix: wrong prop passed to migration fn

kyranjamie committed:
test: mock the bns req response

kyranjamie committed:
fix: refresh on install, closes LEA-2745

kyranjamie committed:
fix: runtime event handler err with collectible cards

Commit:
fix: swaps with undefined route

kyranjamie committed:
fix: legacy auth with missing gaia, closes (leather-io/extension#6302)

kyranjamie committed:
ci: remove job to check signed commits

kyranjamie committed:
Merge pull request (leather-io/extension#6304) from leather-io/release/legacy-auth-fix

Release/legacy auth fix

semantic-release-bot committed:
chore(release): 6.71.0

## [6.71.0](https://github.com/leather-io/extension/compare/v6.70.1...v6.71.0) (2025-06-18)

### Features

* additional inscription controls ([dcda742](https://github.com/leather-io/extension/commit/dcda7425fba7b8af4007724c4ac98ea2a5d849f3))

### Bug Fixes

* legacy auth with missing gaia, closes [(leather-io/extension#6302)](https://github.com/leather-io/extension/issues/6302) ([a82329b](https://github.com/leather-io/extension/commit/a82329b5fc80c0cbaba83ce92b840fe5592482f2))
* refresh on install, closes LEA-2745 ([a97770e](https://github.com/leather-io/extension/commit/a97770ea14c8befda34c17638b82784730a43ad5))
* runtime event handler err with collectible cards ([68adabc](https://github.com/leather-io/extension/commit/68adabc4e9119d7c3fe06f09de67a625de7c91aa))
* stacks tx times, closes LEA-2703 ([f9a0625](https://github.com/leather-io/extension/commit/f9a0625eb6989fd342c940a5dcfdc4e29601d37d))
* swaps with undefined route ([bfe7dc8](https://github.com/leather-io/extension/commit/bfe7dc89afec48189fe8678f45f150eb985211a7))
* test adding switch account to call contract ([4a1c906](https://github.com/leather-io/extension/commit/4a1c906237250109af3312d65ec68652c8db6dc6))
* upgrade to node 22 ([b1d4a4f](https://github.com/leather-io/extension/commit/b1d4a4fb9b6ad3ccad6d57714aec972b8d4a54c3))
* wrong prop passed to migration fn ([168e3c1](https://github.com/leather-io/extension/commit/168e3c12a5c74c2c30e4feb6a2ef48de0bcc5ace))

### Internal

* account signer, closes LEA-2638 ([1d8d662](https://github.com/leather-io/extension/commit/1d8d662a319a23a50993ed76ee9c2decb0dbb6ef))
* add deeper tx analytics ([557bf9a](https://github.com/leather-io/extension/commit/557bf9a9218696f34eeadb672c251c719cbd92ba))
* post-release merge back ([c94b0af](https://github.com/leather-io/extension/commit/c94b0af2e8454bc2653ded68168c11d1aef8571e))
* stx sign transaction to approver, closes LEA-2613 and LEA-2617 ([bae720e](https://github.com/leather-io/extension/commit/bae720ecee9b2ad5a5612d7cd2993c4ab016584a))
* validate send to contract address, closes LEA-2125 ([7d6992f](https://github.com/leather-io/extension/commit/7d6992f27d66be59fb9fe803133948cc33ce7723))

leather-bot committed:
chore: post-release merge back

camerow committed:
ci: mark issues stale after 30 days, close after 14 stale days

kyranjamie committed:
ci: dont comment for newly stale issues

Commit:
refactor: update React again

fabric-8 committed:
fix: update icons

fabric-8 committed:
fix: icon export bug

pete-watters committed:
chore: update webpack-dev-server, ref PR 6288

pete-watters committed:
chore: update prismjs, ref PR 6175

pete-watters committed:
chore: add resolution for pbkdf2 to fix critical vulnerability

pete-watters committed:
chore: add resolution for tar-fs to fix high vulnerability

pete-watters committed:
chore: update webpack https false syntax to specify http

Commit:
fix: downgrade react to 19

Commit:
fix: ststxbtc contract address

Commit:
fix: open swap request double popup

Commit:
feat: promo banner, closes LEA-2978

Commit:
fix: webpack dev server

Commit:
feat: promo bannar tracking, closes LEA-2980

pete-watters committed:
chore(web): update panda-preset and tokens to restore missing borders

kyranjamie committed:
ci: ignore dependabot in all-checks

Commit:
Merge pull request (leather-io/extension#6316) from leather-io/release/sponsored-tx-fixes

release: sponsored tx fixes

semantic-release-bot committed:
chore(release): 6.71.1

## [6.71.1](https://github.com/leather-io/extension/compare/v6.71.0...v6.71.1) (2025-07-15)

### Bug Fixes

* downgrade react to 19 ([dc41d8e](https://github.com/leather-io/extension/commit/dc41d8e16d6e4f14955cbb838fd89120041a3d92))
* icon export bug ([e1dffba](https://github.com/leather-io/extension/commit/e1dffba2b99dbd7527f65f399bb86da96388f448))
* open swap request double popup ([f783c15](https://github.com/leather-io/extension/commit/f783c1503da446c26888f49823236b045280b042))
* rpc flow sbtc sponsorship, closes ENG-97 ([2cd56e4](https://github.com/leather-io/extension/commit/2cd56e41ba61adb45918636b81effefe5eccb3c1))
* rpc sponsored txs, closes ENG-98 ([3ca6a95](https://github.com/leather-io/extension/commit/3ca6a958c224210a36c6fd344067d596491ca96d))
* ststxbtc contract address ([128e558](https://github.com/leather-io/extension/commit/128e558ce0c959a12ad713bde1ebd8db34f00e4c))
* update icons ([fa9ab50](https://github.com/leather-io/extension/commit/fa9ab5079a61458b9c7e5e99bbc2eb9564ba71f4))
* webpack dev server ([387e466](https://github.com/leather-io/extension/commit/387e4661aabeda994ce67d71fbe8e5cceac5f61c))

### Internal

* add back Flag prop removed from lib ([5fc4ea8](https://github.com/leather-io/extension/commit/5fc4ea8cb22abbecc684316d11bf37cced62a892))
* add resolution for pbkdf2 to fix critical vulnerability ([7c1e608](https://github.com/leather-io/extension/commit/7c1e60880136cead794bce9f4d30f2f2ec714bc8))
* add resolution for tar-fs to fix high vulnerability ([6bd9135](https://github.com/leather-io/extension/commit/6bd913540208b3c584600a0d5d96080c28c03fe0))
* post-release merge back ([ca6fc70](https://github.com/leather-io/extension/commit/ca6fc7087ff8e0d09d9c31c3be484d8bbea85588))
* update prismjs, ref PR 6175 ([77a35da](https://github.com/leather-io/extension/commit/77a35daa42ab900d55ce6e07b300f76e3843c2dd))
* update React again ([58e7dcc](https://github.com/leather-io/extension/commit/58e7dcc3f99abcecb487ed190c7757849ee97c90))
* update React to v19, closes LEA-2709 ([5d5d427](https://github.com/leather-io/extension/commit/5d5d427af0833e3d2332b9fb5802c8d3c491e8cd))
* update webpack https false syntax to specify http ([b82e5cf](https://github.com/leather-io/extension/commit/b82e5cfe2cd4d2174c402927776adedc75249a35))
* update webpack-dev-server, ref PR 6288 ([f1be16e](https://github.com/leather-io/extension/commit/f1be16e96ff827fff37ff8fbd29352d93d444c0a))
* upgrade leather packages ([ae8522f](https://github.com/leather-io/extension/commit/ae8522ffdc5e116c139232f39f385e07a56f6322))
* **web:** update panda-preset and tokens to restore missing borders ([2813b80](https://github.com/leather-io/extension/commit/2813b80c1d6b0e050d1043479cdf5557af154ad2))

leather-bot committed:
chore: post-release merge back

tigranpetrossian committed:
refactor: use UI library input in the password field

alexp3y committed:
feat: update bitflow sdk to 2.4

fbwoolf committed:
fix: bitflow gh actions env var

fbwoolf committed:
fix: ledger contract deploy details, closes LEA-3045

tigranpetrossian committed:
refactor: set up currency formatter

tigranpetrossian committed:
refactor: update all formatting-related code to use the new formatter

alexp3y committed:
feat: install leather services package

alexp3y committed:
chore: update leather packages

fbwoolf committed:
fix: missing swap event tracking, closes LEA-3088

alexp3y committed:
feat: implement balance services

tigranpetrossian committed:
refactor: add NBSP helper to test utils, fix a relevant test

alexp3y committed:
fix: share manage token state across accounts

fbwoolf committed:
chore: disable promo banner

alexp3y committed:
chore: bump leather packages

alexp3y committed:
feat: add bitflow env provider address

tigranpetrossian committed:
refactor: migrate Button to new version

alexp3y committed:
chore: update leather packages

kyranjamie committed:
fix: issues introduced with zod upgrade

kyranjamie committed:
fix: use new protections api

tigranpetrossian committed:
fix: use a custom component for fee items

alexp3y committed:
chore: migrate integration tests to testnet4

tigranpetrossian committed:
fix: remove outdated color definitions from sign out flow buttons

tigranpetrossian committed:
fix: use correct balance in the "insufficient balance" message for sip-10 send

kyranjamie committed:
fix: use constant route, divergence introduced loads of bugs

kyranjamie committed:
fix: add analytics to investigate BadNonce errors

kyranjamie committed:
feat: more obvious full page button

In action popup mode only, we now show a header level button to access
fullpage mode. We're tracking both events to see how often users opt to
use this now-obvious feature.

kyranjamie committed:
Merge pull request (leather-io/extension#6348) from leather-io/release/tuesday-eleven-am

semantic-release-bot committed:
chore(release): 6.72.0

## [6.72.0](https://github.com/leather-io/extension/compare/v6.71.1...v6.72.0) (2025-08-27)

### Features

* add bitflow env provider address ([cd8ce4d](https://github.com/leather-io/extension/commit/cd8ce4d7aa629faf6a98642da24491f98318a7f4))
* install leather services package ([371730e](https://github.com/leather-io/extension/commit/371730e566948d6ca563595946acbbde5f438232))
* more obvious full page button ([2fd63c4](https://github.com/leather-io/extension/commit/2fd63c4340c37530bee9e5f32a8b4b708b929ea2))
* promo bannar tracking, closes LEA-2980 ([33c842c](https://github.com/leather-io/extension/commit/33c842cfd00bbcb9f1eeac2d535960d97345c026))
* promo banner, closes LEA-2978 ([1867629](https://github.com/leather-io/extension/commit/1867629a1cf49d9b5a18ce4dda7300afb1ec5c8c))
* update bitflow sdk to 2.4 ([6da3e95](https://github.com/leather-io/extension/commit/6da3e95fe98132743432e67b9f9f305002746aa3))

### Bug Fixes

* add analytics to investigate BadNonce errors ([3817403](https://github.com/leather-io/extension/commit/38174036af4063cf9f49bd60d1ff3892e97d4f59))
* bitflow gh actions env var ([d2745d6](https://github.com/leather-io/extension/commit/d2745d6418193b3a9e102593dd2481633a237223))
* issues introduced with zod upgrade ([d47043e](https://github.com/leather-io/extension/commit/d47043e107241a7a477e057225e4999faf60ec4f))
* ledger contract deploy details, closes LEA-3045 ([8792e8b](https://github.com/leather-io/extension/commit/8792e8bc1bd9e8a380fb9ec7e5fed86450004f67))
* missing swap event tracking, closes LEA-3088 ([9683573](https://github.com/leather-io/extension/commit/9683573a50df380f84f0007aea6ca534b5317b33))
* remove outdated color definitions from sign out flow buttons ([129983d](https://github.com/leather-io/extension/commit/129983d6b9174563aae5c772f5a84789a37c0880))
* share manage token state across accounts ([199a192](https://github.com/leather-io/extension/commit/199a192b53c70b20f347c6fcce768fa3d044e2f8))
* use a custom component for fee items ([3bfcd8f](https://github.com/leather-io/extension/commit/3bfcd8f8050a1b9b96b2d99726c363d1b201b95a))
* use constant route, divergence introduced loads of bugs ([2c7a1aa](https://github.com/leather-io/extension/commit/2c7a1aad66b5903e0a3d632e380d4d2261438818))
* use correct balance in the "insufficient balance" message for sip-10 send ([5e18eaa](https://github.com/leather-io/extension/commit/5e18eaa1a17d1ef74b7f491d0f42600347d46bda))
* use new protections api ([d44161c](https://github.com/leather-io/extension/commit/d44161ce6189ee1fdc34667e874fe21464d86a97))

### Internal

* add NBSP helper to test utils, fix a relevant test ([580add8](https://github.com/leather-io/extension/commit/580add88932eb8d52875b035a434c43ca2386b9c))
* bump leather packages ([f90010a](https://github.com/leather-io/extension/commit/f90010af1c0b8e2d5b63e3c2cc4aebb3582df5c8))
* disable promo banner ([7c84980](https://github.com/leather-io/extension/commit/7c84980d38f24aa97983eac10c033e07399c9e95))
* migrate Button to new version ([424fb60](https://github.com/leather-io/extension/commit/424fb605b0cec3231fccb5bf2eb3658f2ebd158c))
* migrate integration tests to testnet4 ([267945e](https://github.com/leather-io/extension/commit/267945e0fbeda93ac54664fb64bda153f22094df))
* post-release merge back ([1fc6809](https://github.com/leather-io/extension/commit/1fc6809063bfdb9b2370fc7495d2baa35d1a93fa))
* set up currency formatter ([88caed6](https://github.com/leather-io/extension/commit/88caed6e2d640c540892e6ba3d6b8c46419b1145))
* update all formatting-related code to use the new formatter ([0d2f4a9](https://github.com/leather-io/extension/commit/0d2f4a90b31d8a06312f81cb71ad94cf42fec126))
* update leather packages ([be82613](https://github.com/leather-io/extension/commit/be82613879ac5126e2467463a785671f4eea0a43))
* update leather packages ([c0465fa](https://github.com/leather-io/extension/commit/c0465fa1448eb641d17bbde8f80f036c8cec6263))
* use UI library input in the password field ([9ec2888](https://github.com/leather-io/extension/commit/9ec28884a8c8e64a7bfbd408dce33cd1fdd1d4f8))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix: add api nonce values to report

kyranjamie committed:
fix: add fullscreen btn to lock screen

kyranjamie committed:
ci: optimised all checks pass job

kyranjamie committed:
fix: rpc events etc should not allow fullscreen

kyranjamie committed:
fix: stamp schema fails

kyranjamie committed:
fix: accessible button markup

kyranjamie committed:
fix: advanced menu in settings dropdown

kyranjamie committed:
Merge pull request (leather-io/extension#6350) from leather-io/release/wednesday

semantic-release-bot committed:
chore(release): 6.72.1

## [6.72.1](https://github.com/leather-io/extension/compare/v6.72.0...v6.72.1) (2025-08-28)

### Bug Fixes

* accessible button markup ([e81fe5c](https://github.com/leather-io/extension/commit/e81fe5c01037687c6302c7f6075a3f7ffa402db5))
* add api nonce values to report ([deb3a53](https://github.com/leather-io/extension/commit/deb3a53e2921b42e82489c33ddd87e0fcd903692))
* add fullscreen btn to lock screen ([39c4b97](https://github.com/leather-io/extension/commit/39c4b972192a091dc19e47533e7467a9b5fc392d))
* advanced menu in settings dropdown ([7a3b3e9](https://github.com/leather-io/extension/commit/7a3b3e93cc4484e718b7a1e16386bfac1c80db6c))
* rpc events etc should not allow fullscreen ([d71e519](https://github.com/leather-io/extension/commit/d71e519d6b07b15400419c5b192c590ede6a246e))
* stamp schema fails ([7abeefc](https://github.com/leather-io/extension/commit/7abeefcb8b12eec0c62e78cb9151c5d21b4fdb16))

### Internal

* post-release merge back ([bcfed33](https://github.com/leather-io/extension/commit/bcfed334b2412def8623469360195d31929f5382))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix: report the actual unnsigned tx nonce

kyranjamie committed:
fix: upgrade tanstack query

kyranjamie committed:
fix: stale nonce problem

kyranjamie committed:
Merge pull request (leather-io/extension#6354) from leather-io/release/fix-nonce

semantic-release-bot committed:
chore(release): 6.72.2

## [6.72.2](https://github.com/leather-io/extension/compare/v6.72.1...v6.72.2) (2025-08-29)

### Bug Fixes

* report the actual unnsigned tx nonce ([25ee4a2](https://github.com/leather-io/extension/commit/25ee4a2b3defff592a71d9a83a7ffe833c48100f))
* stale nonce problem ([835dab4](https://github.com/leather-io/extension/commit/835dab4c78da42a7be88e1c0b5fed50c892cbcb6))
* upgrade tanstack query ([f61c7d7](https://github.com/leather-io/extension/commit/f61c7d743e5f4b64f4d109ec14c7c6ee91d5ea09))

### Internal

* post-release merge back ([5c03d92](https://github.com/leather-io/extension/commit/5c03d927e8a0f40b8ac49f108c6d5b7a57903837))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
feat: track session duration

kyranjamie committed:
feat(ledger): add windows firmware warning

kyranjamie committed:
Merge pull request (leather-io/extension#6359) from leather-io/release/mountain-air

semantic-release-bot committed:
chore(release): 6.73.0

## [6.73.0](https://github.com/leather-io/extension/compare/v6.72.2...v6.73.0) (2025-09-09)

### Features

* **ledger:** add windows firmware warning ([fd4419b](https://github.com/leather-io/extension/commit/fd4419bffb4158c853337ebb9362abad852a98a3))
* track session duration ([bcc8e96](https://github.com/leather-io/extension/commit/bcc8e96dca6060e798cda3617cc7885940cfeb25))

### Internal

* post-release merge back ([f497ed5](https://github.com/leather-io/extension/commit/f497ed59f5d8defb725978a17ce68f269be15921))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix: sbtc analytics

kyranjamie committed:
fix(ledger): sbtc deposit

kyranjamie committed:
fix(ledger): sbtc incompatibility

alexp3y committed:
fix(ledger): sbtc routing

kyranjamie committed:
fix(swap): max swap amount

kyranjamie commented:
If possible, I wonder if it'd make most sense to take only the last 2 years' worth of commits, to avoid bringing over too much worthless history

edgarkhanzadian commented:
@kyranjamie sure, i can squash starting from some X commit. Which one do you think should be that X commit?

edgarkhanzadian committed:
feat: prepare extension for monorepo

kyranjamie committed:
Merge pull request (leather-io/extension#6363) from leather-io/release/sbtc-improvements

Release/sbtc improvements

semantic-release-bot committed:
chore(release): 6.73.1

## [6.73.1](https://github.com/leather-io/extension/compare/v6.73.0...v6.73.1) (2025-09-15)

### Bug Fixes

* **ledger:** sbtc deposit ([a8ea282](https://github.com/leather-io/extension/commit/a8ea282661f45c4704d6e98f8119862f9a6c5227))
* **ledger:** sbtc incompatibility ([a66ff36](https://github.com/leather-io/extension/commit/a66ff365bc03e1b3026c2c69a3b8bc4aff05e841))
* **ledger:** sbtc routing ([730e68a](https://github.com/leather-io/extension/commit/730e68a3de51744539596172f09089e47a3728f1))
* sbtc analytics ([45844cd](https://github.com/leather-io/extension/commit/45844cdd17586f1629854601be2ea1e98d7a7355))
* **swap:** max swap amount ([c7a4ff7](https://github.com/leather-io/extension/commit/c7a4ff7d8e7cd3ed315fecba01c8a69981479b2c))

### Internal

* post-release merge back ([a691950](https://github.com/leather-io/extension/commit/a69195050fb086c5cc5c6e001e2ecb19af83cf16))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix: remove old sbtc withdrawal warning

kyranjamie committed:
refactor: react 19.1

kyranjamie committed:
Merge pull request (leather-io/extension#6367) from leather-io/release/tuesday

semantic-release-bot committed:
chore(release): 6.73.2

## [6.73.2](https://github.com/leather-io/extension/compare/v6.73.1...v6.73.2) (2025-09-16)

### Bug Fixes

* remove old sbtc withdrawal warning ([d3df67a](https://github.com/leather-io/extension/commit/d3df67ad1ac6af47deaf053f1691a0b6a9ffddeb))

### Internal

* post-release merge back ([4955e5a](https://github.com/leather-io/extension/commit/4955e5abb0b123708982050fb25ab8e8e20a72a9))
* react 19.1 ([90fa5e7](https://github.com/leather-io/extension/commit/90fa5e73102fd677ffd27bd05fce4af2d7dfedff))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix(sbtc): use leather client for signer key fetch

kyranjamie committed:
fix(sbtc): navigate to error page

kyranjamie committed:
fix: nested button error

kyranjamie committed:
fix(sbtc): passing non-finalized tx

kyranjamie committed:
Merge pull request (leather-io/extension#6371) from leather-io/release/ledger-sbtc-fix

semantic-release-bot committed:
chore(release): 6.73.3

## [6.73.3](https://github.com/leather-io/extension/compare/v6.73.2...v6.73.3) (2025-09-17)

### Bug Fixes

* nested button error ([fc229b9](https://github.com/leather-io/extension/commit/fc229b9298f9817bfc23f3ec56eb51b73099b336))
* **sbtc:** navigate to error page ([276f245](https://github.com/leather-io/extension/commit/276f24513227bea50b7397eac962bc5f81d6d6b5))
* **sbtc:** passing non-finalized tx ([f2b07a3](https://github.com/leather-io/extension/commit/f2b07a38ed9732117f4b4ab0426ffc7d32f6d5f0))
* **sbtc:** use leather client for signer key fetch ([7713669](https://github.com/leather-io/extension/commit/771366948d38d49ecc47943a5b2ccef97eb2ea23))

### Internal

* post-release merge back ([5400198](https://github.com/leather-io/extension/commit/54001985a40d98044570e6c5edee0ad73cac85ee))

leather-bot committed:
chore: post-release merge back

kyranjamie committed:
fix(analytics): cap session duration tracking

kyranjamie committed:
chore: enable promo banner

kyranjamie committed:
fix: rephase promo cards

kyranjamie committed:
refactor: stacks transaction summary LEA-3200

kyranjamie committed:
fix: missing resp data check

alexp3y committed:
fix: bignumber shiftedby error

alexp3y committed:
feat: apply visibility services

kyranjamie committed:
fix: remove console.log

kyranjamie committed:
Merge pull request (leather-io/extension#6380) from leather-io/release/monday-migrate

Release/monday migrate

semantic-release-bot committed:
chore(release): 6.73.4

## [6.73.4](https://github.com/leather-io/extension/compare/v6.73.3...v6.73.4) (2025-10-06)

### Bug Fixes

* **analytics:** cap session duration tracking ([f34c15d](https://github.com/leather-io/extension/commit/f34c15de0b3df2e94ef57460503337d72b67c751))
* missing resp data check ([f37a287](https://github.com/leather-io/extension/commit/f37a28706d23d172a671cc6f2e92cc9bda9e3697))
* remove console.log ([eb64532](https://github.com/leather-io/extension/commit/eb6453292a852e2f321e45f816e605ebfe9ef7ea))
* rephase promo cards ([69446cd](https://github.com/leather-io/extension/commit/69446cd7560636816b270bdd8f723343846793cc))

### Internal

* enable promo banner ([d5c5f49](https://github.com/leather-io/extension/commit/d5c5f49e5acb5e40e44e4a04f5d5241e4371a0fc))
* post-release merge back ([67655f0](https://github.com/leather-io/extension/commit/67655f0b0d54f024b1f652799a320395fe1eb0fb))
* stacks transaction summary LEA-3200 ([18697ce](https://github.com/leather-io/extension/commit/18697ce920609a39b7dea21e4568237b6b1a8c1f))

leather-bot committed:
chore: post-release merge back

edgarkhanzadian committed:
fix: update ui package

edgarkhanzadian committed:
fix: revert pnpm-lock

edgarkhanzadian committed:
fix: update @leather.io/services to 1.34.2

kyranjamie committed:
fix: update all packages

kyranjamie committed:
fix: replace all in tx status badge

alexp3y committed:
fix: sip10 send asset name

edgarkhanzadian committed:
feat(extension): leather-io/extension into feat/extension-move
